react/legacy/index.js

```diff
@@ -134,6 +138,12 @@ Header.schema = {
       default: true,
       isLayout: true,
     },
+    showLocaleSwitcher: {
```

Comment:
I want able to see this configuration in the storefront, can you take a look into that?